contracts/ccip/ccip_offramp/sources/offramp.move

```diff
@@ -265,8 +265,8 @@ fun init(_witness: OFFRAMP, ctx: &mut TxContext) {
         off_ramp_object_id: object::id_address(&off_ramp_object),
     };
 
-    let tn = type_name::with_original_ids<OFFRAMP>();
-    let package_bytes = ascii::into_bytes(tn.address_string());
+    let tn = type_name::get_with_original_ids<OFFRAMP>();
```

Comment:
i think `get_with_original_ids` and `get_address` are deprecated.

Reply:
Will revert.